plugins/dac_data_manager.c

```diff
@@ -1927,7 +1927,7 @@ static int dds_dac_init(struct dac_data_manager *manager,
 	manager->dacs_count++;
 	ddac->index = manager->dacs_count;
 
-	return ret;
+	return 0;
```

Comment:
As this api stands now, there's no point in returning anything... I'm not seeing a path where an error is set. Hence, I'm also not seeing the point of the `fprintf()`...

In fact, I'm wondering if we should have a better way of detecting such devices... Like this, we loose the ability to say that something is wrong for devices that are capable of more than 1 tone

Reply:
yes, you are right that we need a better way of detecting devices that do not have DDS.
One criteria would be that the number of tones is equal to 0.

Reply:
Thank you for pointing this out!
I made some changes. Do you think the new variant is more suitable?